src/adapters/index.ts

```diff
@@ -91,16 +102,20 @@ export interface EmitterAdapter {
   /**
   Subscribe to incoming events
 
-  @param name - A string identifying the type of event that this subscription should consume. This
-  should look something like `accounts.ProfileUpdated`, `organisations.MembershipUpdated` or similar
+  @param ns - A string identifying the **namespace** of the event that this subscription should
+  consume. This should look something like `accounts` or `organisations`.
+
+  @param ty - A string identifying the **type** of the event that this subscription should consume.
+  This should look something like `LoggedIn` or `OrganisationUpdated`.
 
   @param handler - An {@link EmitterHandler} function called when the incoming event has been saved
   successfully. Any domain-specific logic like sending a notification or triggering some data
   processing should be performed here.
   */
   subscribe<T extends Event<EventData, EventContext<any>>>(
-    name: T["data"]["type"],
-    handler: EmitterHandler<T>,
+    ns: T["data"]["event_namespace"],
```

Comment:
Can we use a more descriptive names for the arguments? `event_namespace` and `event_type` are not that verbose if we take into consideration that as a user of the library you do not have to type them.